plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -12,9 +12,12 @@
 {-# LANGUAGE NPlusKPatterns        #-}
```

Comment:
Nothing to do with this PR, but isn't this supposed to be a terrible thing to do?

Reply:
It's not that terrible, just ancient. I guess I was trying to be close to the typed CEK machine where we have matching in `mkBuiltinApplication`. I'll remove that once I implement the alternative builtlin application machinery.